build.yaml

```diff
@@ -1,8 +1,13 @@
 targets:
   $default:
     builders:
+      build_cli:
+        generate_for:
+          - lib/src/cli/metrics/options.dart
+      json_serializable:
+        generate_for:
+          - lib/src/cli/metrics/flutter_info.dart
```

Comment:
is this required work by the user

Reply:
No. We need it.

Reply:
thanks, I see. If its a dev dependency its fine. please ignore my comment.